### DISCUSSION TITLE
下载安装整合包出现问题

### DISCUSSION THREAD

**Discussion by LeoLeo1125** [Meloong-Git/PCL#1045] [提问] [answered]
整合包安装失败
→ 构造 Mod 下载信息失败
→ 无法将类型为“Newtonsoft.Json.Linq.JValue”的对象强制转换为类型“Newtonsoft.Json.Linq.JArray”。
   在 PCL.ModModpack._Closure$__6-0._Lambda$__2(LoaderTask`2 Task)
   在 PCL.ModLoader.LoaderTask`2._Closure$__12-0._Lambda$__0()
错误类型：System.InvalidCastException

**Comment by WForst-Breeze** [comment, marked as answer]
您是否在尝试下载 ATM8 整合包时出现此问题？如是，请见 #1006，此问题已于最新快照版修复，如您是正式版使用者，请耐心等待，谢谢~
